lib/generators/stimulus_reflex/templates/config/initializers/stimulus_reflex.rb

```diff
@@ -17,4 +17,10 @@
   # eg. if your connection is `identified_by :current_user` and your User model has an email attribute, you can access r.email (it will display `-` if the user isn't logged in)
 
   # config.logging = proc { "[#{session_id}] #{operation_counter.magenta} #{reflex_info.green} -> #{selector.cyan} via #{mode} Morph (#{operation.yellow})" }
+
+  # Set `enable_middleware_processing` to `true` if StimulusReflex should process
+  # the Rails middleware stack before executing the reflex. This could be useful
+  # if you do request rewriting in your rack middleware.
+
+  # config.enable_middleware_processing = false
```

Comment:
Maybe nit-picking but not sure about the name that has `enable` prefix, since it will looks strange on getters such as `if config.enable_middleware_processing`, for me `middleware_processing` or `middeware_procesing_enabled` reads better

Reply:
What about `config.process_middleware = true/false`? That gives you `if config.processmiddleware` which reads nicely.

Reply:
Thanks! I update the name to `middleware_processing` 🙌🏼